ReactNative/ExampleSuperApp/ios/ExampleSuperApp.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,856 @@
+		C9E747942A35504400D3F898 /* GXDataLayerRemote.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = GXDataLayerRemote.framework; path = "../../../../../Library/Developer/Xcode/DerivedData/GXFrameworks-behydqtgtzwwlnamlgkbycqxuaem/Build/Products/Debug-iphonesimulator/GXDataLayerRemote.framework"; sourceTree = "<group>"; };
+		C9E747952A35504400D3F898 /* GXCoreBL.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = GXCoreBL.framework; path = "../../../../../Library/Developer/Xcode/DerivedData/GXFrameworks-behydqtgtzwwlnamlgkbycqxuaem/Build/Products/Debug-iphonesimulator/GXCoreBL.framework"; sourceTree = "<group>"; };
+		C9E747962A35504500D3F898 /* GXSuperApp.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = GXSuperApp.framework; path = "../../../../../Library/Developer/Xcode/DerivedData/GXFrameworks-behydqtgtzwwlnamlgkbycqxuaem/Build/Products/Debug-iphonesimulator/GXSuperApp.framework"; sourceTree = "<group>"; };
+		C9E747972A35504500D3F898 /* YAJL.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = YAJL.framework; path = "../../../../../Library/Developer/Xcode/DerivedData/GXFrameworks-behydqtgtzwwlnamlgkbycqxuaem/Build/Products/Debug-iphonesimulator/YAJL.framework"; sourceTree = "<group>"; };
+		C9E747982A35504500D3F898 /* GXObjectsModel.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = GXObjectsModel.framework; path = "../../../../../Library/Developer/Xcode/DerivedData/GXFrameworks-behydqtgtzwwlnamlgkbycqxuaem/Build/Products/Debug-iphonesimulator/GXObjectsModel.framework"; sourceTree = "<group>"; };
+		C9E747992A35504500D3F898 /* GXStandardClasses.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = GXStandardClasses.framework; path = "../../../../../Library/Developer/Xcode/DerivedData/GXFrameworks-behydqtgtzwwlnamlgkbycqxuaem/Build/Products/Debug-iphonesimulator/GXStandardClasses.framework"; sourceTree = "<group>"; };
+		C9E7479A2A35504500D3F898 /* GXUIApplication.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = GXUIApplication.framework; path = "../../../../../Library/Developer/Xcode/DerivedData/GXFrameworks-behydqtgtzwwlnamlgkbycqxuaem/Build/Products/Debug-iphonesimulator/GXUIApplication.framework"; sourceTree = "<group>"; };
+		C9E7479B2A35504500D3F898 /* GXFoundation.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = GXFoundation.framework; path = "../../../../../Library/Developer/Xcode/DerivedData/GXFrameworks-behydqtgtzwwlnamlgkbycqxuaem/Build/Products/Debug-iphonesimulator/GXFoundation.framework"; sourceTree = "<group>"; };
```

Comment:
These relative paths are not correct. Should point to [iOS/GeneXus Frameworks](https://github.com/genexus-books/gx-super-app/tree/main/iOS/GeneXus%20Frameworks)